docs/userGuide/siteJsonFile.md

```diff
@@ -19,6 +19,7 @@ Here is a typical `site.json` file:
   "faviconPath": "myfavicon.png",
   "titlePrefix": "FooBar Dev Docs",
   "theme": "bootswatch-cerulean",
+  "highlightStyle": "light",
```

Comment:
should we do 

```
style: {
  codeTheme: "..."
  theme: ... // later pr
}
```

instead? so we can move `theme` inside there later on since there's now multiple style properties

Reply:
`highlightStyle` sounds a little like the highlight text style we have, https://deploy-preview-1355--markbind-master.netlify.app/userguide/formattingcontents#text-styles, should we change this to something more explicit?

Reply:
`style` looks more apt to consolidate all the style configurations for the site (I don't know how it will be in the future, we only have two style properties so far). I'll introduce that property, and also rename `highlightStyle` to `codeTheme`.

Reply:
Also, a quick question, what do you think should happen when authors write an ineligible option to `codeTheme` (e.g. `"qwerty"`)? Should MarkBind halt or let it quietly discard the option and continue to use the default?

Reply:
hmm haven't thought of validating site.json before, my preference here is to `logger.warn` the user softly and fall back to the dark theme since style isn't critical

for other properties like `baseUrl` that are we could throw exceptions (in a later pr perhaps)

Reply:
if you're interested, you could add validation for all applicable properties together in a later pr as well, and just leave this one be for now :-)